plugins/woocommerce/src/Blocks/BlockTypes/MiniCart.php

```diff
@@ -595,14 +676,23 @@ protected function get_template_part_contents( $template_name = 'mini-cart' ) {
 		$template_part = get_block_template( $template_slug_to_load . '//' . $template_name, 'wp_template_part' );
 
 		if ( $template_part && ! empty( $template_part->content ) ) {
-			$template_part_contents = do_blocks( $template_part->content );
+			if ( $do_blocks ) {
+				$template_part_contents = do_blocks( $template_part->content );
+			} else {
```

Comment:
I'd add a comment here that we want to process the content before applying do_blocks; 

> For now, I'm going to leave it like this, but it's something that will have to be changed when the switch to the new block is made.

@luisherranz How do you usually keep track of things that need changing? Should we add a comment with the next steps, or what needs to be changed?

Reply:
As this is an obvious change, I wasn't planning to document it, since whoever does the final PR to change the old implementation to the new one will have to review this part of the code and will see that it no longer makes sense to pass the `$do_blocks` variable.

For other changes and technical debt related to the Interactivity API, @santosguillamot and I have started to document it internally, but if there is a standard way to do it in WooCommerce, please let us know and we will adapt.